auacm/app/static/html/editCompetition.html

```diff
@@ -53,27 +53,25 @@ <h1 class="pull-left">{{ title }}</h1>
       </div>
 
       <div class="checkbox">
-        <label><input type="checkbox" value="" ng-model="competitionClosed">Closed competition (Only admins can register users)</label>
+        <label>
+          <input type="checkbox" value="" ng-model="competitionClosed">
+          Closed competition (Only admins can register users)
+        </label>
       </div>
     </form>
-
   </div>
-</div>
-<div class="panel panel-default">
-  <div class="panel-heading">
-    <div class="panel-title">
-      Problems
-    </div>
-  </div>
-  <div class="panel-body">
-    <table class="table table-bordered">
-      <thead>
-        <tr>
-          <th class="col-sm-4">Label</th>
-          <th class="col-sm-4">Name</th>
-          <th class="col-sm-4"></th> <!-- This is reserved for the add/remove button -->
-        </tr>
-      </thead>
+
+  <h3 class="col-md-12">Problems</h3>
+
+  <table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="col-sm-4">Label</th>
+        <th class="col-sm-4">Name</th>
+        <th class="col-sm-4"></th> <!-- This is reserved for the add/remove button -->
+      </tr>
+    </thead>
+    <tbody>
```

Comment:
I agree. This looks weird, but for some reason, the tables just don't get top borders. How do you feel about changing it to `table-striped` instead of `table-bordered`? I think it looks pretty good.